client/src/components/KeyboardShortcutDialog.vue

```diff
@@ -21,20 +21,13 @@ export default defineComponent({
           ['z', 'x'],
         ],
       ],
+      ['Place crosshairs at location', [['click']]],
       ['Toggle fullscreen', [['e', 'd', 'c']]],
-      ['Zooming', [['right button + dragging']]],
+      ['Changing window width & level', [['click + dragging']]],
+      ['Zooming', [['right button + dragging'], ['Ctrl + dragging'], ['Alt + dragging']]],
       ['Panning', [['shift + dragging']]],
       ['Cancel on confirm dialog', [['esc']]],
-      ['Increase/decrease window', [['=', '-']]],
-      ['Increase/decrease window level', [['[', ']']]],
-      ['Focus to note', [['n']]],
-      ['Show note history', [['h']]],
-      ['Unfocus from note', [['esc']]],
-      ['Mark as bad/good/usable extra', [['b', 'g', 'u']]],
-      ['Save', [['alt + s']]],
-      ['Save on confirm dialog', [['y']]],
-      ["Don't save on confirm dialog", [['n']]],
-      ['Cancel on confirm dialog', [['esc']]],
```

Comment:
Were all of these buttons unbound? I don't remember doing that

Reply:
Filed issue #375 to reimplement more keyboard shortcuts later